mod-source-record-storage-server/src/main/java/org/folio/consumers/AuthorityLinkChunkKafkaHandler.java

```diff
@@ -87,13 +87,14 @@ public AuthorityLinkChunkKafkaHandler(RecordService recordService, KafkaConfig k
 
   @Override
   public Future<String> handle(KafkaConsumerRecord<String, String> consumerRecord) {
-    LOGGER.trace("handle:: Handling kafka record: {}", consumerRecord);
```

Comment:
Please return this log message.

Reply:
This change was debugging purpose and it would be reverted at the end.